test/Microsoft.AspNetCore.Authentication.Test/CookieTests.cs

```diff
@@ -1247,7 +1247,7 @@ private TestServer CreateServer(Action<CookieAuthenticationOptions> configureOpt
             => CreateServerWithServices(s =>
             {
                 s.AddSingleton<ISystemClock>(_clock);
-                s.AddAuthentication().AddCookie(configureOptions);
+                s.AddAuthentication(o => o.DefaultScheme = CookieAuthenticationDefaults.AuthenticationScheme).AddCookie(configureOptions);
```

Comment:
An `s.AddAuthentication(CookieAuthenticationDefaults.AuthenticationScheme)` overload for this would be nice.